.clang-format

```diff
@@ -16,7 +16,7 @@ FixNamespaceComments: false
 IncludeCategories:
   - Regex:           '^<'
     Priority:        4
-  - Regex:           '^"(llvm|llvm-c|clang|clang-c)/'
+  - Regex:           '^"(llvm|llvm-c|clang|clang-c|fir|mlir|mlir-c)/'
```

Comment:
Shouldn't fir be at the same priority as flang? It's in this (sub-)project but the rest listed here are not.

Reply:
Until yesterday, f18 didn't use an include structure like llvm.

Reply:
Could we roll a discussion about these clang-format files into #945? It would be nice to have a cohesive clang-format file for the whole F18 project.

Reply:
The second reason `fir` is grouped with `mlir` which is grouped with `llvm` is that `fir` is a dialect of `mlir` and makes no sense without `mlir` (which transitively makes no sense without `llvm`). Conversely, the FIR dialect has no dependence on the f18 front-end and could be targeted by other Fortran front-ends.

Agreed with David that debating .clang-format issues should be waged in its own forum.

Reply:
As requested, discussion moved to: https://github.com/flang-compiler/f18/pull/945#issuecomment-580184803
Please comment directly there if you wish to reply to this thread.